satpy/modifiers/angles.py

```diff
@@ -266,7 +266,8 @@ def _sanitize_observer_look_args(*args):
             new_args.append(STATIC_EARTH_INERTIAL_DATETIME)
         elif isinstance(arg, (float, np.float64, np.float32)):
             # round floating point numbers to nearest tenth
-            new_args.append(round(arg, 1))
+            # Hashing numpy float types doesn't seem to work:
+            new_args.append(float(round(arg, 1)))
```

Comment:
This is not part of the solution I suggested. Let's discuss in the related issue...